modules/virt-about-ssp-operator.adoc

```diff
@@ -6,7 +6,7 @@
 [id="virt-about-ssp-operator_{context}"]
 = About the Scheduling, Scale, and Performance (SSP) Operator
 
-The SSP Operator, `ssp-operator`, deploys the common templates, the related default boot sources, the pipeline tasks, and the template validator.
+The SSP Operator, `ssp-operator`, deploys the default boot sources, the pipeline tasks, and the template validator.
 
```

Comment:
```suggestion
The SSP Operator, `ssp-operator`, deploys the common templates, the related default boot sources, and the template validator.
```
`common templates` and `common instancetypes` are two different things.

The `common templates` stays in ssp-operator. Looks like the `common instancetypes` was not yet documented here.

The ` the pipeline tasks` should be removed, too. https://issues.redhat.com/browse/CNV-29468